lib/paranoia.rb

```diff
@@ -1,7 +1,7 @@
 require 'active_record' unless defined? ActiveRecord
 
 if [ActiveRecord::VERSION::MAJOR, ActiveRecord::VERSION::MINOR] == [5, 2] ||
-   ActiveRecord::VERSION::MAJOR > 5
+   ActiveRecord::VERSION::MAJOR >= 6
   require 'paranoia/active_record_5_2'
```

Comment:
looking at this change, this looks badly name... do we need this patch for rails 6, 7 and 8 ?